<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>473</width>
        <height>510</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="generalGroup">
         <property name="title">
          <string>Morrowind Content Language</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="languageComboBox">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="wizardGroup">
         <property name="title">
          <string>Morrowind Installation Wizard</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
          <item row="1" column="0">
           <widget class="QPushButton" name="wizardButton">
            <property name="text">
             <string>Run &amp;Installation Wizard</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="importerGroup">
         <property name="title">
          <string>Morrowind Settings Importer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="1,0">
            <item row="0" column="0">
             <widget class="QLabel" name="importerLabel">
              <property name="text">
               <string>File to import settings from:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="settingsComboBox"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="browseButton">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="addonsCheckBox">
            <property name="text">
             <string>Import add-on and plugin selection (creates a new Content List)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0">
            <item row="2" column="0">
             <widget class="QPushButton" name="importerButton">
              <property name="text">
               <string>Run &amp;Settings Importer</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="maximum">
             <number>4</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="testingGroup">
         <property name="title">
          <string>Testing</string>
         </property>
         <layout class="QVBoxLayout" name="testingGroupVerticalLayout">
          <item>
           <widget class="QCheckBox" name="skipMenuCheckBox">
            <property name="text">
             <string>Skip menu and generate default character</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="startDefaultCharacterAtHorizontalLayout">
            <item>
             <spacer name="startDefaultCharacterAtHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="startDefaultCharacterAtLabel">
              <property name="text">
               <string>Start default character at</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="startDefaultCharacterAtField">
              <property name="placeholderText">
               <string>default cell</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Run script after startup:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="runScriptAfterStartupHorizontalLayout">
            <item>
             <widget class="QLineEdit" name="runScriptAfterStartupField"/>
            </item>
            <item>
             <widget class="QPushButton" name="runScriptAfterStartupBrowseButton">
              <property name="text">
               <string>Browseâ€¦</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
